src/main/java/practice/StreamPractice.java

```diff
@@ -14,7 +16,13 @@ public class StreamPractice {
      * "Can't get min value from list: < Here is our input 'numbers' >"
      */
     public int findMinEvenNumber(List<String> numbers) {
-        return 0;
+
+        return numbers.stream().map(e -> e.split(","))
```

Comment:
For the future, don't use short names, fix them for yourself
All magic numbers, strings, and symbols in your code should be constants.
[this article](https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names)
```suggestion
        return numbers.stream().map(line -> line.split(SEPARATOR))
```
It looks more readable